src/ParameterBag/MailRecipientParameterBag.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * This file is part of the NotifierMail package.
+ *
+ * (c) Dries De Peuter <dries@nousefreak.be>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Notifier\Mail\ParameterBag;
+
+use Notifier\Mail\MailChannel;
+use Notifier\ParameterBag\ParameterBagInterface;
+
+/**
+ * @author Joeri van Dooren
+ */
+class MailRecipientParameterBag implements ParameterBagInterface
+{
+    /**
+     * @var string
+     */
+    private $to;
```

Comment:
Can you indicate what the function of $to is. Indicating it is an email will make the implementation more clear.